lib/Doctrine/ORM/AbstractQuery.php

```diff
@@ -346,10 +346,9 @@ static function (Query\Parameter $parameter) use ($key): bool {
      * Sets a collection of query parameters.
      *
      * @param ArrayCollection|mixed[] $parameters
+     * @psalm-param ArrayCollection<int, Parameter>|mixed[] $parameters
```

Comment:
It would be great to be able to automate the following, can you research how to do this (if that's indeed doable?):
```suggestion
     * @param       ArrayCollection|mixed[]                 $parameters
     * @psalm-param ArrayCollection<int, Parameter>|mixed[] $parameters
```

Reply:
What I see from the tests phpcs actually forces the opposite. At least removing the spaces between type and parameter.